packages/example-app-base/src/components/SiteMessageBar/SiteMessageBar.tsx

```diff
@@ -1,33 +1,13 @@
 import * as React from 'react';
 import { Link } from 'office-ui-fabric-react/lib/Link';
-import { MessageBar, IMessageBarStyles, IMessageBarStyleProps } from 'office-ui-fabric-react/lib/MessageBar';
-import { IStyleFunctionOrObject } from 'office-ui-fabric-react/lib/Utilities';
-
-export interface ISiteMessageBarProps {
-  /** Text for the message bar */
-  text: string;
-
-  /** Text for a link shown after the main text */
-  linkText?: string;
-
-  /** URL for a link shown after the main text */
-  linkUrl?: string;
-
-  /**
-   * Prefix for session storage key for storing if the message bar has been closed.
-   * If not provided, local storage will not be used.
-   */
-  sessionStoragePrefix?: string;
-
-  /** Styles for the message bar */
-  styles?: IStyleFunctionOrObject<IMessageBarStyleProps, IMessageBarStyles>;
-}
+import { MessageBar } from 'office-ui-fabric-react/lib/MessageBar';
```

Comment:
If this is going to be an independent component, I don't think it should be called SiteMessageBar. I thought about calling it just MessageBar instead, but having component names that duplicate built-in Fabric components isn't great. Maybe Banner would work?

Reply:
I don't like the idea of having duplicate names either (our Table components, ugh). SiteMessage or SiteMessageBar make the most sense in my opinion, but maybe it should stick to the fabric-website directory so its purpose is more explicit?